src/test/java/integration/DiscountPerCounterpartyControllerIT.java

```diff
@@ -118,18 +115,18 @@ public void createDiscountPerCounterparty() throws Exception {
                         statusCode(SC_OK).
                 extract().
                         path("uuid");
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
         UUID newDiscountUuid = UUID.fromString(newUuid);
 
         DiscountPerCounterparty createdDiscountPerCounterparty = discountPerCounterpartyService.
                 getEntityByUuid(newDiscountUuid, user);
         DiscountPerCounterpartyDto discountPerCounterpartyDto = discountPerCounterpartyMapper.
                 toDto(createdDiscountPerCounterparty);
-        long timeCreated = createdDiscountPerCounterparty.getCreated().getTime();
-        long timeModified = createdDiscountPerCounterparty.getLastModified().getTime();
+        LocalDateTime timeCreated = createdDiscountPerCounterparty.getCreated();
+        LocalDateTime timeModified = createdDiscountPerCounterparty.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).